docs/source/models.rst

```diff
@@ -219,20 +219,20 @@ EfficientNet-B4                   83.384          96.594
 EfficientNet-B5                   83.444          96.628
 EfficientNet-B6                   84.008          96.916
 EfficientNet-B7                   84.122          96.908
-regnet_x_400mf                    72.834          90.950
-regnet_x_800mf                    75.212          92.348
-regnet_x_1_6gf                    77.040          93.440
-regnet_x_3_2gf                    78.364          93.992
-regnet_x_8gf                      79.344          94.686 
-regnet_x_16gf                     80.058          94.944
-regnet_x_32gf                     80.622          95.248
-regnet_y_400mf                    74.046          91.716
-regnet_y_800mf                    76.420          93.136
-regnet_y_1_6gf                    77.950          93.966
-regnet_y_3_2gf                    78.948          94.576
-regnet_y_8gf                      80.032          95.048
-regnet_y_16gf                     80.424          95.240
-regnet_y_32gf                     80.878          95.340
+regnet_x_400mf                    74.864          92.322
+regnet_x_800mf                    77.522          93.826
+regnet_x_1_6gf                    79.668          94.922
+regnet_x_3_2gf                    81.196          95.430
+regnet_x_8gf                      81.682          95.678
+regnet_x_16gf                     82.716          96.196
+regnet_x_32gf                     83.014          96.288
+regnet_y_400mf                    75.804          92.742
+regnet_y_800mf                    78.828          94.502
+regnet_y_1_6gf                    80.876          95.444
+regnet_y_3_2gf                    81.982          95.972
+regnet_y_8gf                      82.828          96.330
+regnet_y_16gf                     82.886          96.328
+regnet_y_32gf                     83.368          96.498
```

Comment:
@kazhang This is a non-blocking comment for FYI and discussion.

We haven't updated these numbers for any other model that we refreshed. The reason is that the overall documentation of this section seems to imply that these accuracies are achieved by using the `pretrained=True` API. Since the old API gives you access to the old weights, we didn't change these numbers. This is debatable. Perhaps we could change the phrasing and update it. Or perhaps we should keep the numbers as-is for now. Thoughts?

The plan is to revamp the documentation of models in H1 with @NicolasHug and programmatically build the docs from the meta-data stored in the weights.